### PR TITLE
Milestone 2: Works and About Me section

### PR DESCRIPTION
**Pull request by AbeeraTahir**
In this PR of the mobile-first design of the Portfolio project, I have added:

1. Works section using CSS Grid and Flexbox
2. About myself section using Flexbox
3. A section for programming languages, frameworks, and skills.
4. Button interactions (hover and active)

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -170,3 +170,528 @@ nav {
...
+  border: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+}
```
Comment (zieeco) [inline style.css:348]: - Please, kindly ensure you've added the interactivity on the button to indicate the hover, active, enabled, and disabled state on the button to improve the user experience below:

button

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @AbeeraTahir,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (AbeeraTahir) [reply style.css:348]: Hi @zieeco,

I have added all these states on the button, but implementing the button enabled is canceling the hover and active effect. What should I do now?

File: index.html [hunk 2]
```diff
@@ -42,5 +42,238 @@ <h2 class="supporting_text">
             </ul>
         <div class="headline_bg_img"></div>
     </section>
```
Comment (jerryowusu) [inline index.html:44]: The issue with the background image is till pending. I can see you made an effort to fix it but there is an empty white space on the right when you stretch it as shown below. Kindly check and fix it.

Screenshot from 2022-09-24 01-31-19

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @AbeeraTahir 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

soclose

 ## Highlights
:heavy_check_mark: All linters are passing
:heavy_check_mark: Good commit messages

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AbeeraTahir** [comment]
Hi @jerryowusu,

Thank you for your review but I didn't get your point that how it is still causing empty space on the right because now, I can see that there is no scroll bar when I resize it according to the mobile screen. I also added a border just to confirm if it still fits in the mobile view screen or not. Can you give a suggestion? Cause I am stuck.

Please elaborate further as it is my last review. Thank you.

**Review by emmiiorji — APPROVED** [review]
Hi @AbeeraTahir, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

congratulations


## Good Highlights✔

- Linter checks are passing
- Good commit messages
- Requested changes have been implemented


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**